github/ci/prow-deploy/files/jobs/kubevirt/macvtap-cni/macvtap-cni-presubmits.yaml

```diff
@@ -13,15 +13,18 @@ presubmits:
         grace_period: 5m
       max_concurrency: 6
       labels:
-        preset-dind-enabled: "true"
+        preset-podman-in-container-enabled: "true"
         preset-docker-mirror-proxy: "true"
         preset-shared-images: "true"
       cluster: ibm-prow-jobs
       spec:
         containers:
-          - image: quay.io/kubevirtci/bootstrap:v20220728-1410a63
+          - image: quay.io/kubevirtci/bootstrap:v20220823-3fed276
             securityContext:
               privileged: true
+            env:
+            - name: OCI_BIN
+              value: podman
```

Comment:
Does this make sense ?... AFAIU, this is for the unit test lane, which doesn't need containers.

Reply:
possible that you are right, i just augmented them both without classification
lets try to remove it (i guess you prefer)

Reply:
Thank you; if it goes wrong, add it back - we can investigate later.

Reply:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2118/rehearsal-pull-macvtap-cni-unit-test/1562340373840269312

passed